### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by kennisilverio**
added knex migration and seed files along with implementing an async …  …
…await function on the seed files and fixed the scripts to run the migrations and seed files

### CODE REVIEW

**Comment by kennisilverio** [comment]
@dhpatel15 @stacy-kim @zpei100 please review and approve when possible

File: db/seeds/development/seedCategories.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const faker = require('faker');
+
+exports.seed = async function(knex, Promise) {
```
Comment (stacy-kim) [inline db/seeds/development/seedCategories.js:3]: Nice use of async here!

File: knexfile.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
```
Comment (stacy-kim) [inline knexfile.js:6]: Are you using sqlite3 or postgresql?

**Review by stacy-kim — APPROVED** [review]
I think you left something about sqlite3 in one of the files. Otherwise, it's very clean and easy to read. Good job!

**Pull request merged**